release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2606,6 +2606,11 @@ link:https://access.redhat.com/solutions/5812591[{product-title} 4.6.18 containe
 
 With this update, the Insights Operator now collects information for `ContainerRuntimeConfig` objects. This information is useful for troubleshooting. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1891544[*BZ#1891544*].
 
+[id="ocp-4-6-18-cco-cred-rotation"]
+===== Support for rotating cloud provider credentials
+
+With this update, you can manually update the secret that the Cloud Credential Operator (CCO) uses to manage cloud provider credentials. For more information, see xref:../post_installation_configuration/cluster-tasks.adoc#manually-rotating-cloud-creds_post-install-cluster-tasks[Rotating cloud provider credentials manually].
```

Comment:
Is it redundant or repetitive to have _update_ twice in the sentence, especially where they mean two different things? Wonder if it will cause translation issues (likely not)?

Reply:
Will change to "with this release" for less repetition :)